src/plugins/intel_cpu/src/nodes/rnn.cpp

```diff
@@ -388,6 +391,10 @@ void RNN::fillCellDesc() {
         inCandidate.emplace_back(std::make_shared<DnnlBlockedMemoryDesc>(WShape, memory::data_type::f32, memory::format_tag::nc));
         inCandidate.emplace_back(std::make_shared<DnnlBlockedMemoryDesc>(RShape, memory::data_type::f32, memory::format_tag::nc));
         inCandidate.emplace_back(std::make_shared<DnnlBlockedMemoryDesc>(BShape, memory::data_type::f32, memory::format_tag::x));
+
+        if (cell_type == dnnl::algorithm::vanilla_lstm && P != 0) {
+            inCandidate.emplace_back(std::make_shared<DnnlBlockedMemoryDesc>(PShape, memory::data_type::f32, memory::format_tag::x));
```

Comment:
Does it mean that onednn actually support Peephole modification of LSTM?
Should we specify this input in the current version of this operation?

Reply:
oneDNN does support LSTM with peephole but it is not enabled in CPU plugin.
We do not have any tests for LSTM peephole.
I do not think we had any request to enable it.

Also this change does not really enable it.

Worth to mention, that there is no information about peephole in ngraph documentation
@dmitry-gorokhov I think the enablement should be done in scope of separate activity, what do you think?

Reply:
@avoskoboinyk-lohika Will it fix the current conformance test issue if CPU plugin will just return "unsupported" for LSTM with peephole instead of crash?

Reply:
AFAIK peephole input was only supported by LSTMCell-0 and removed in newer version. Current MO generates LSTMCell-4 and I am not aware about real models/use cases which required peephole support. In other words there is no any value in LSTMCell-0 with peephole support on plugin level.
I agree with @EgorDuplensky that the plugin should just return false in `isSupportedOperation` for such case.

Reply:
Done